src/components/timeline/index.js

```diff
@@ -227,22 +286,31 @@ class Timeline extends React.PureComponent<Props> {
 }
 
 export default explicitConnect<{||}, StateProps, DispatchProps>({
-  mapStateToProps: state => ({
-    globalTracks: getGlobalTracks(state),
-    globalTrackOrder: getGlobalTrackOrder(state),
-    globalTrackReferences: getGlobalTrackReferences(state),
-    committedRange: getCommittedRange(state),
-    zeroAt: getZeroAt(state),
-    panelLayoutGeneration: getPanelLayoutGeneration(state),
-    timelineType: getTimelineType(state),
-    hiddenTrackCount: getHiddenTrackCount(state),
-  }),
+  mapStateToProps: state => {
+    const configuration = getProfilerConfiguration(state);
+    const activeBrowsingContextID = configuration
+      ? configuration.activeBrowsingContextID || null
```

Comment:
This is a dangerous use of `||` with an integer: is it possible that its value is `0` ?

Reply:
`0` means we failed to get the BrowsingContext ID from the browser. The job of this `||`  is precisely convert `0` to `null`. Adding a comment that explains this.

Reply:
Yes, I added a comment to this code(and moved this to a selector). Please see `src/selectors/profile.js` file. And sorry about the outdated codes after some commits. I wanted to rebase everything to appropriate commits but there were a lot of changes, and it was really hard to make :/